lib/fedex/tracking_information.rb

```diff
@@ -44,6 +44,10 @@ def initialize(details = {})
         @delivery_at = Time.parse(details[:actual_delivery_timestamp])
       end
 
+      if details.has_key?(:estimated_delivery_timestamp)
+        @estimated_delivery_at = Time.parse(details[:estimated_delivery_timestamp])
```

Comment:
yeah it is UTC
the only thing is, the `estimated_delivery_timestamp` key isn't always present in the payload. and none of the tests pass for me locally in lib/fedex/track_spec.rb :/